BabelfishFeatures.cfg

```diff
@@ -611,6 +611,8 @@ supported-1.0.0=arg4=NO_SUPPORTED_ARGUMENTS_RIGHT_NOW
 
 [ALTER VIEW]

```

Comment:
Has it been determined that no Java changes are required? My understanding was that there were cases which would not be supported , and Compass should detect those.